doc/CONTRIBUTING.md

```diff
@@ -337,6 +337,34 @@ for more configuration details. Everything within `python_config` is imported
 via `dictConfig` (in addition to some standard settings defined in
 `dataactcore.logging`.
 
+### Setup with Docker
+
+Install docker in your local machine by selecting your OS and hitting install from this [link](https://docs.docker.com/docker-for-mac/install).
```

Comment:
Does docker still only work on Mac or is this a way people with Windows can also join us? If it's the latter, do we want to link differently? If it's the former, ignore this comment.

Reply:
If using docker, this should work in windows also. I updated the link to the generic install link.